### PR TITLE
Document device twin config example for AU915RP1 region

### PR DESCRIPTION
**Pull request by ouphi**
# PR for issue #1923

## What is being addressed
Document device twin config example for AU915RP1 region.

### CODE REVIEW

**Comment by ouphi** [comment]
Wait for the PR about supporting AU915 to be merged before merging this doc.

File: docs/issues.md
Comment (Mandur) [inline docs/issues.md:14]: You can also add AS 923 and CN 470 here

**Review by Mandur — APPROVED** [review]

**Pull request merged**
